jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -103,10 +113,15 @@ def create(template: str, params: Dict, custom_resource_dir: Optional[str] = Non
         try:
             utils.create_from_yaml(__k8s_clients.k8s_client, path)
         except FailToCreateError as e:
-            if e.api_exceptions[0].status == 409:
-                print('exists already')
-            else:
-                raise e
+            for api_exception in e.api_exceptions:
+                if api_exception.status == 409:
+                    # The exception's body is the error response from the
+                    # Kubernetes apiserver, it looks like:
+                    # {..."message": "<resource> <name> already exists"...}
+                    resp = json.loads(api_exception.body)
+                    logger.info(f'🔁\t{resp["message"]}')
```

Comment:
shouldn't use a warning log to highlight this exception message?

```suggestion
                    logger.warning(f'🔁\t{resp["message"]}')
```